week-4/Homework/mandatory/4-groceries.js

```diff
@@ -28,7 +28,9 @@ Exercise 1:
   Then use console.log() to print out the list.
 */
 // Gather all week item names into this array
-let weeklyGroceriesToBuy = [];
+
+//let weeklyGroceriesToBuy = Object.keys(weeklyMealPlan);
+//weeklyGroceriesToBuy.forEach(daysOfWeek => console.log(`${weeklyMealPlan[daysOfWeek]}`));
```

Comment:
You were on the right sort of lines here with this one. Maybe consider iterating using `Object.values()` instead of `Object.keys()` and think about how you can update the `weeklyGroceriesToBuy` array